frontend/pages/train.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import Container from 'react-bootstrap/Container';
+
+import TrainAlmondForm from 'components/TrainAlmondForm';
+
+export default props => {
+  return (
+    <Container fluid>
+      <h1>Train Almond</h1>
+      <p>
+        If Almond is misbehaving or misinterpreting your input, you can correct
+        it here.
```

Comment:
Content on all pages (with the possible exception of the /about pages) need to be translatable. I don't know how people commonly translate react. We have been using gettext so far.

Reply:
Will look into it. There are react translation packages